notebooks/idr0062_prediction.ipynb

```diff
@@ -454,6 +481,25 @@
     "conn.close()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "d2b111fe",
+   "metadata": {},
+   "source": [
+    "## Exercises\n",
+    "\n",
+    "**Exercise 1:**\n",
+    " - Read the geojson file.\n",
+    " - Display the shapes on a selected z-section e.g. z = 5.\n",
+    "\n",
+    "**Exercise 2:**\n",
+    " - Convert the Stardist labels into OMERO polygons.\n",
+    " - Save the converted labels to an OMERO.server if the image is not coming from a public instance.\n",
```

Comment:
```suggestion
    " - Save the converted labels to an OMERO.server if possible.\n",
```

the reason for the simplification suggested is that the present sentence might be confusing. The image is ultimately coming from the public instance, and there are several implicit steps which are not explained anywhere. So I would leave it out and simply indicate in some cases this action is not possible.

Reply:
We need to indicate how exactly to connect to a non-idr read-write server, otherwise the user will execute on the idr and get an error.